release/release.go

```diff
@@ -135,6 +135,10 @@ func isWSL() bool {
 	return false
 }
 
+// PreseedMode states whether snapd is running in a presseding mode
+// where some functions and subsystems should alter their behavior.
```

Comment:
// ... to perform a partial first boot and touch only filesystem state inside a chroot.